### ISSUE TITLE
Prototype: two-line match row vs the 6.D density lock (A17)

### ISSUE THREAD

**Issue by soniCaH** [soniCaH/www.kcvvelewijt.be#2599]
## The problem

`AgendaMatchRow` renders `{home} — {away}` on a single line with `truncate`. Two long Belgian club names destroy it on a phone: "K Hoger Op Wolvertem Merchtem" against anything leaves the name clipped mid-word, and the reader cannot tell who is playing.

## The candidate

Sofascore's stacked row — home over away, score in a right-hand column — is immune by construction. It is also arguably **more** fanzine, not less: a printed results grid stacks home over away with the score in a right-hand column. That is the register we are already in.

## Why this needs a prototype rather than a decision

The stacked row **doubles row height**, and the 6.D "labelled wall" density lock was a considered decision, not an accident. A research document should not overturn a design lock on instinct.

Instinct going in (to be tested, not assumed): **responsive** — stacked below `sm`, single-line above. That keeps desktop density and fixes the phone case, but it means two layouts to maintain and a reflow at the breakpoint.

## What the prototype should compare

Build the real thing, with real data, and put the variants side by side:

1. **Current** — single line, truncated (the baseline, so the cost is visible)
2. **Stacked always** — home over away, score right column
3. **Responsive** — stacked below `sm`, single line above

Test each against the cases that actually break:

- [ ] Two long names ("K Hoger Op Wolvertem Merchtem" vs "Ritterklub Vsv Jette")
- [ ] Short names ("KCVV Elewijt" vs "SK Laar") — does stacking look wasteful when it is not needed?
- [ ] A result (score present) **and** a fixture (kick-off time instead of score)
- [ ] The `A` suffix that PSD appends to team names
- [ ] 320px, 390px, and desktop
- [ ] A full band of rows, not one row in isolation — density is a property of the wall, not the brick

## Constraints (locked, do not propose otherwise)

- Sharp corners — no border-radius anywhere
- Press-down hover (`translate-x-1 translate-y-1 shadow-none`)
- No new fonts — Freight Sans Pro body, freight-big-pro display, plus mono
- Cream paper + jersey-deep; bright jersey green is banned as a large surface
- Photos stay in colour
- No emojis — Phosphor Fill icons
- Motion respects `prefers-reduced-motion`

## Dependency

**Blocked by / interacts with #2598 (homepage horizontal overflow).** Cause 2 in that issue (`w-full` inside the card's flex row) lives in this exact component. If the stacked row wins here, that markup changes anyway.

Sequence: land cause 1 of #2598 independently, prototype this against a card that is **not** already overflowing, then fix cause 2 in whichever markup survives — so the card is restructured once rather than twice.

## Source

A17 in `docs/research/decision-sheet.md` §3, arising from `docs/research/mobbin-interaction-patterns.md`.

**Comment by soniCaH** [comment]
> *This was generated by AI during a human-queue session.*

## Owner decision, 2026-09-21: **home over away on phones, win/loss colours the whole score box, no THUIS/UIT marker**

Decided in three rounds against screenshots of the **live** `/kalender` (real fonts, real data, all 77 September rows), rebuilt in the browser per option. Comparison page + screenshots land with the build under `docs/design/mockups/2599-match-row/`.

### Today is worse than this issue says

A phone opens `/kalender` in the agenda list by default (`CalendarWidget.tsx` `useIsPhoneViewport`), so **every** phone visitor gets `AgendaMatchRow`:

| | names cut | what you see |
| --- | --- | --- |
| 320px | 77 / 77 rows | name slot **0–14px** — `F..`, `K`, `l`, or nothing |
| 390px | 76 / 77 rows | name slot ~84px — mostly `KCVV Ele…`, i.e. our own club |

### The decision — below `sm` only; desktop does not change

```text
09:15   [crest] FC Zemst Sportief   ┌5┐
U10     [crest] KCVV Elewijt        └3┘   ← box tinted: brick = lost
```

1. **Left margin:** kick-off time, and under it our squad label (`U10`, `A-ploeg`, `U8 Wit`). There is no team filter on `/kalender`, so scanning one edge for your child's squad is the row's main job.
2. **Middle:** both clubs stacked, **home on top**, each with its crest, one line each.
3. **Right:** the score stacked per club, one number per line.
4. **Win/loss = the whole score box tinted** — the same `OUTCOME_UNDERLINE.light` tints (jersey-deep won, brick lost, none for a draw), filling the box behind both numbers instead of an underline. It sits in the same place on every row.
5. **No `THUIS`/`UIT` marker.** The order says it — the rule the desktop `TeamAgendaRow` already follows (`detail-ia-locked.md`: *"No venue indicator (the order conveys home/away)"*). The team page only adds its House/Bus icon on phones because it shows the opponent alone there.

Measured: 320px names cut **24 / 154** lines (was 77 / 77 rows), 390px **3 / 154**. Row 42px → ~68px; a 14-match Saturday 591px → ~958px.

### Rejected — and why, so they are not re-proposed

- **One line, opponent alone** (the smallest change): does **not** fix 320px — time + squad + score + `THUIS` still take the whole row, names get 0–14px (72 / 77 cut).
- **Two lines, opponent alone, squad in the margin, score over `THUIS`/`UIT`:** reads names well, but a result needs decoding (`5 – 3` + `UIT` = we scored 3), and the middle holds one line in a two-line row. Home over away uses that space for the second club and reads the result directly.
- **Win/loss underline under our own number:** jumps between the top and bottom line row to row. Looked wrong.
- **No win/loss colour:** would be the only score on the site without it.
- **House/Bus icon instead of `THUIS`/`UIT`:** says the order twice; the team page's own rule puts a marker only where the order is missing.
- **The 6.D "density lock" is not broken:** 6.D chose *show every row, never fold a busy day* over folding (`6d2-agenda-density-compare.html`). It never ruled on row height.

### Folded in — same row

- **The squad label leaks the federation prefix:** `KCVVE U8 Wit` / `KCVVE U8 Groen` on `/kalender`, `U8 Wit` / `U8 Groen` on `/ploegen`. One team, one name (#2630). The label comes from `teamsWithPsd[i].displayName` in `kalender/page.tsx`.

## Build checklist

- [ ] `AgendaMatchRow` (`CalendarAgenda.tsx`): below `sm`, the stacked layout above; `sm`+ keeps today's one-line row untouched.
- [ ] Left column wide enough for `U8 Groen` / `Reserven` on one line (~56px, not 52px).
- [ ] Score box: stacked per-club numbers, whole-box tint from `OUTCOME_UNDERLINE.light` (same `isSettledMatch` gate as today); no box for an unplayed match.
- [ ] No `MatchVenueTag` below `sm`.
- [ ] `ReservationAgendaRow`: same left column (time over squad) so the margin reads down the whole day.
- [ ] Squad label without the `KCVVE ` prefix, matching `/ploegen`.
- [ ] Update the `CalendarAgenda` stories; capture VR for the consumer stories.
- [ ] Commit the comparison page + screenshots to `docs/design/mockups/2599-match-row/`.
